test/tests/ha/rackdh_ha_resource_test.py

```diff
@@ -0,0 +1,120 @@
+"""
+Copyright 2017, Dell, Inc.

```

Comment:
Update the copyright, "Copyright (c) 2017 Dell Inc. or its subsidiaries.  All Rights Reserved."
I need to fix the template!